### PR TITLE
minor docs updates

### PR DESCRIPTION
**Pull request by wesinator**
#5431 new PR

### CODE REVIEW

**Comment by norg** [comment]
You could run `clang-format` on the files you worked on so you can ensure the format is correct.

File: doc/userguide/rules/intro.rst [hunk 1]
```diff
@@ -276,6 +276,7 @@ Some keywords function act as modifiers. There are two types of modifiers.
 
   In the above example the pattern 'index.php' is modified to inspect the HTTP uri buffer.
 
+
```
Comment (jasonish) [inline doc/userguide/rules/intro.rst:279]: I don't see this extra line changing the output appearance at all.  Is there another purpose to it?

Comment (wesinator) [reply doc/userguide/rules/intro.rst:279]: ok, I thought it would add whitespace between the items. 
No other purpose

**Comment by wesinator** [comment]
#5900

**Pull request closed without merging**